website/docs/r/schema_site_anp_epg_domain.html.markdown

```diff
@@ -38,18 +41,48 @@ resource "mso_schema_site_anp_epg_domain" "site_anp_epg_domain" {
 
 ```
 
+### domain_dn usage ###
+
+```hcl
+resource "mso_schema_site_anp_epg_domain" "site_anp_epg_domain" {
+  schema_id = "5c4d9fca270000a101f8094a"
+  template_name = "Template1"
+  site_id = "5c7c95b25100008f01c1ee3c"
+  anp_name = "ANP"
+  epg_name = "Web"
+  domain_dn = "uni/vmmp-VMware/dom-VMware-ab"
+  deploy_immediacy = "lazy"
+  resolution_immediacy = "lazy"
+  vlan_encap_mode = "static"
+  allow_micro_segmentation = true
+  switching_mode = "native"
+  switch_type = "default"
+  micro_seg_vlan_type = "vlan"
+  micro_seg_vlan = 46
+  port_encap_vlan_type = "vlan"
+  port_encap_vlan = 45
+  enhanced_lag_policy_name = "name"
+  enhanced_lag_policy_dn = "uni/vmmp-VMware/dom-VMware-ab/vswitchpolcont/enlacplagp-Lacp"
+
+}
+
+```
+
 ## Argument Reference ##
 
 * `schema_id` - (Required) SchemaID under which you want to deploy Anp Epg Domain.
 * `template_name` - (Required) Template where Anp Epg Domain to be created.
 * `site_id` - (Required) SiteID under which you want to deploy Anp Epg Domain.
 * `anp_name` - (Required) Name of Application Network Profiles.
 * `epg_name` - (Required) Name of Endpoint Group to manage.
-* `dn` - (Required) The domain profile name.
+* `dn` - (Optional) **Deprecated**. The domain profile name. This is required when `domain_dn` is not used. This attribute requires `domain_type` and `vmm_domain_type` (when it is applicable) to be set.
```

Comment:
I would add that domain_name or domain_dn should be used instead. (similar to the error message above)